### ISSUE TITLE
Aggregator save_polygons fails when there is an empty polygon

### ISSUE THREAD

**Issue by melisandeteng** [hugobaudchon/geodataset#86]
Before applying self.save_polygons(), empty polygons should be removed from self.polygons_gdf   otherwise a segmentation fault will occur. 
would it be possible to add a couple lines checking and removing empty polygons from the self.polygons_gdf dataframe? Thanks! 

(see call for save_polygons here https://github.com/hugobaudchon/geodataset/blob/389f3c82af22878939589d75e0fceb94009f596e/geodataset/aggregator/aggregator.py#L51)

**Comment by hugobaudchon** [comment]
Absolutely, I've added some additional checks, they will be published in v0.2.3.
